mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,21 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
+function getMood(bool) {
+  let isHappy = bool;
+
 
   if (isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
-
+//we need to call a function 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num;
 
-  if (isBigEnough) {
+  if (isBigEnough > 10) {
```

Comment:
`isBigEnough` implies either yes, it is, or no, it isn't, i.e. a boolean. So we should set this variable to 
```
const isBigEnough = num > 10
```
And then in the if statement we only need to pass `if (isBigEnough)`. You solution works but the variable name doesn't explain properly what the variable is representing since you just set it to number.